app/gateway/changelog.md

```diff
@@ -11,6 +11,32 @@ For Kong Gateway OSS, view the [OSS changelog on GitHub](https://github.com/Kong
 
 For product versions that have reached the end of sunset support, see the [changelog archives](https://legacy-gateway--kongdocs.netlify.app/enterprise/changelog/).
 
+## 3.7.1.2
+**Release Date** 2024/07/09
+
+### Deprecations
+
+* Debian 10, CentOS 7, and RHEL 7 reached their End of Life (EOL) dates on June 30, 2024. 
+As of this patch, Kong is not building Kong Gateway 3.7.x installation packages or Docker images for these operating systems.
+As well, Kong is no longer providing official support for any Kong version running on these systems.
+
+### Features
+
+#### Plugins
+
+* [**AWS Lambda**](/hub/kong-inc/aws-lambda) (`aws-lambda`)
+  * Added the new configuration parameter `empty_arrays_mode`, which lets you control whether Kong Gateway should send 
+  empty arrays (`[]`) returned by the Lambda function as empty arrays (`[]`), or as empty objects (`{}`) in JSON responses.
+
+### Fixed
+
+* Fixed an issue where the Dev Portal documentation link was unavailable because the official documentation was removed after the 3.4.x.
```

Comment:
```suggestion
* Fixed an issue where the Dev Portal documentation link was unavailable because the official documentation was removed after  3.4.x.
```